openshift-examples/keycloak-https-mutual-tls.json

```diff
@@ -185,6 +192,7 @@
                             {
                                 "name": "${APPLICATION_NAME}",
                                 "image": "jboss/keycloak-openshift",
+                                "args": ["-c", "standalone-ha.xml"],
```

Comment:
Shouldn't changing to standalone-ha.xml be handled by the image itself rather than having to change the args here?

Reply:
Ok, so I had a pretty long chat with Infinispan folks. In short - the replicated caches (used in `standalone-ha.xml`) will start slower (since they try to do discovery) but the runtime performance should be roughly the same as local ones. 

So it makes sense to always start the HA profile. I will add necessary switches.

Reply:
Done.